images/cfw/etc/init.d/S99crontab

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+
+DEVICE_SN=$(cat /proc/cmdline | xargs -n1 | grep "bbl_serial=" | sed "s|bbl_serial=||")
```

Comment:
This will work, though this shenanigan is needed only at initial boot before we have most of the security service started.  I think the more idiomatic way to do this after filesystems are mounted is `$(bbl_3dpsn)`.

Reply:
I see other places also check `/oem/device/sn` before calling `bbl_3dpsn`. Should I include that here as well?

Reply:
Those are only necessary for the install/kexec environment, which needs to be compatible with old base firmwares that may not have a working `bbl_3dpsn`.  I think `bbl_3dpsn` is good enough here.